spec/factories/questions.rb

```diff
@@ -0,0 +1,11 @@
+FactoryGirl.define do
+  factory :question do
+    sequence(:body) { |n| "Question #{n}" }
+
+    trait :with_answers do
+      after :create do |question, evaluator|
```

Comment:
Unused block argument - `evaluator`. If it's necessary, use `_` or `_evaluator` as an argument name to indicate that it won't be used.